samples/bot-all-cards/csharp/README.md

```diff
@@ -0,0 +1,174 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send different types of cards using bot.
+products:
+- office-teams
+- office
+- office-teams
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "21/10/2022 07:54:21 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-all-cards-csharp
+
+---
+
+# Types of Cards
+
+This sample shows the feature where user can send different types of cards using bot.
+
+## Interaction with app
+
+![all-cards-sample ](BotAllCards/Images/allBotCardsGif.gif)
+
```

Comment:
We need to add one more section for demo manifest of C# code:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/636/files#diff-7bb681f0cc7d221ebbf0b17dacc3a1b81bf3bb250a4770a3162eaee026e62e57

Reply:
Fixed